pages/genre/_itemId/index.vue

```diff
@@ -70,26 +80,16 @@ export default Vue.extend({
   },
   async beforeMount() {
     this.setAppBarOpacity({ opaqueAppBar: true });
+    this.setPageTitle({
+      title: this.genre.Name
+    });
 
-    const type = this.$route.query.type;
     try {
-      this.genre = (
-        await this.$api.userLibrary.getItem({
-          userId: this.$auth.user?.Id,
-          itemId: this.$route.params.itemId
-        })
-      ).data;
-
-      this.setPageTitle({
-        title: this.genre.Name
-      });
-
       this.items = (
         await this.$api.items.getItems({
-          uId: this.$auth.user?.Id,
           userId: this.$auth.user?.Id,
-          genreIds: this.$route.params.itemId,
-          includeItemTypes: type,
+          genreIds: [this.$route.params.itemId],
+          includeItemTypes: [this.$route.query.type.toString()],
           recursive: true,
           sortBy: 'SortName',
           sortOrder: 'Ascending'
```

Comment:
Shouldn't this be in the asyncData function?

Reply:
Yes, but currently this takes a while with large libraries.

To prevent it making the experience miserable, I left it there for now. I'll likely move it to `fetch()` in a followup PR.

Reply:
Ok, makes sennse